content/en/docs/usage/make_build.md

```diff
@@ -0,0 +1,352 @@
+
+Booting from ROM..Powered by
+o.   .o       _ _               __ _
+Oo   Oo  ___ (_) | __ __  __ _ ' _) :_
+oO   oO ' _ `| | |/ /  _)' _` | |_|  _)
+oOo oOO| | | | |   (| | | (_) |  _) :_
+ OoOoO ._, ._:_:_,\_._,  .__,_:_, \___)
+                 Phoebe 0.10.0~ee2a37cf
+Hello world!
+```
+
+If you are working on a machine without full virtualisation support, the `-enable-kvm` option can be left out.
+You can also add the `-W` flag to the `qemu-guest` script.
+This will lower the performance, since everything will be emulated.
+
+```console
+$ qemu-system-x86_64 -enable-kvm -cpu host -nographic -kernel build/app-helloworld_kvm-x86_64
+SeaBIOS (version 1.13.0-1ubuntu1.1)
+
+iPXE (http://ipxe.org) 00:03.0 CA00 PCI2.10 PnP PMM+07F8C8A0+07ECC8A0 CA00
+
+Booting from ROM..Powered by
+o.   .o       _ _               __ _
+Oo   Oo  ___ (_) | __ __  __ _ ' _) :_
+oO   oO ' _ `| | |/ /  _)' _` | |_|  _)
+oOo oOO| | | | |   (| | | (_) |  _) :_
+ OoOoO ._, ._:_:_,\_._,  .__,_:_, \___)
+                 Phoebe 0.10.0~ee2a37cf
+Hello world!
+
+$ ../../unikraft/support/scripts/qemu-guest -W -k build/app-helloworld_kvm-x86_64
+SeaBIOS (version 1.13.0-1ubuntu1.1)
+
+iPXE (http://ipxe.org) 00:03.0 CA00 PCI2.10 PnP PMM+07F8C8A0+07ECC8A0 CA00
+
+Booting from ROM..Powered by
+o.   .o       _ _               __ _
+Oo   Oo  ___ (_) | __ __  __ _ ' _) :_
+oO   oO ' _ `| | |/ /  _)' _` | |_|  _)
+oOo oOO| | | | |   (| | | (_) |  _) :_
+ OoOoO ._, ._:_:_,\_._,  .__,_:_, \___)
+                 Phoebe 0.10.0~ee2a37cf
+Hello world!
+```
+
+See [here](docs/operations/plats/kvm/#build-and-run-unikraft-on-kvm-using-make-and-qemu) for further information on running the unikernel with `KVM`.
+
+### Clean the Build Files
+
+To remove the build files of the application, we can run:
+
+```console
+$ make clean
+make[1]: Entering directory '/home/stefan/.unikraft/unikraft'
+  CLEAN   libkvmplat
+  CLEAN   libkvmpci
+  CLEAN   libkvmvirtio
+[...]
+```
+
+This will keep the fetched files and delete everything else was generated during the build step.
+If we want to also remove the fetched files we can use:
+
+```console
+$ make properclean
+make[1]: Entering directory '/home/stefan/.unikraft/unikraft'
+  RM      build/
+make[1]: Leaving directory '/home/stefan/.unikraft/unikraft'
+```
+
+We can also remove the generated configuration files by using:
+
+```console
+$ make distclean
+make[1]: Entering directory '/home/stefan/.unikraft/unikraft'
+  RM      build/
+  RM      config
+make[1]: Leaving directory '/home/stefan/.unikraft/unikraft'
+```
+
+## Extending the Build
+
+The build above was the simplest one we can do with Unikraft - configuring, building and running app-helloworld.
+Often, we want to extend the build:
+
+* Add external libraries (e.g. [lwip](https://github.com/unikraft/lib-lwip), [musl](https://github.com/unikraft/lib-musl)).
+* Add new source files to the build.
+* Use different configuration options.
+
```

Comment:
Add subsections for the above three extension steps. Using triple-hash (`### ...`).